src/sentry/relay/projectconfig_cache/base.py

```diff
@@ -10,8 +10,8 @@ def __init__(self, **options):
     def set_many(self, configs):
         pass
 
-    def delete_many(self, project_ids):
+    def delete_many(self, public_keys):
```

Comment:
This is just a variable rename right? We still pass the same thing in?

Reply:
yes, we pass public keys in